### ISSUE TITLE
Lastest release of descartes (1.3.2) is not compatible with the most recent version of pitest (1.17.2)

### ISSUE THREAD

**Issue by Zialus** [STAMP-project/pitest-descartes#149]
#### Description

The most recent release of the plugin doesn't work with the latest pitest

#### Steps to reproduce

Execute `mvn verify` in a project with pitest 1.17.2

**Comment by oscarlvp** [comment]
Thank you for reporting the issue. An update will be available in the next weeks.

**Comment by Zialus** [comment]
Any updates on this @oscarlvp ?

**Comment by oscarlvp** [comment]
Yes. I'm still working on this. I think I found the issue: a change in the way PIT identifies plugins and a provided dependency that is not available anymore. There are some tests to do using Gradle and the command line to check the plugin also works outside Maven with the new changes and also some refactoring that is ongoing.
